.taskcat.yml

```diff
@@ -23,6 +23,8 @@ project:
     QSS3BucketName: $[taskcat_autobucket]
     QSS3BucketRegion: $[taskcat_current_region]
   regions:
+    #! Don't change that without adapting the internal pipeline, we use the
+    #! first .project.region as the default for CI buckets (for caching, ...)
```

Comment:
This file also lives in the upstream repo. Troy (AWS) once asked me to add more regions including govCloud as their CI can do this.

Reply:
Yeah. We should and can do that. But there is still stuff we need to do, as per discussion on the other PR. @leclaist will handle that. 

Maybe the wording is off here; what I mean to say is: please don't change the zeroth element in the list without changing the internal pipeline or moving some internal buckets around; we use the zeroth element to determine the location of some random internal CI buckets (e.g. tap-quickstart). You can add as many additional regions as you want, tho.